### ISSUE TITLE
SD Maid fails with busybox error

### ISSUE THREAD

**Issue by elhennig** [d4rken/sdmaid-public#376]
Hi,
SD maid (4.1.6) started to fail on almost every task (even settings) on my Samsung Galaxy S5 mini. I guess it is a permission error, but I did not find out where to fix it on my own. 
Any hint is appreciated...
[log.txt](https://github.com/d4rken/sdmaid-public/files/228006/log.txt)

**Comment by d4rken** [comment]
It should work perfectly on an S5 mini, a busybox error on that device indicates permission problems while SD Maid tries to do its setup.

It's either a current problem with root access, or a previous problem that resulted in wrong permissions.

Remove SD Maid, make sure `/data/data/eu.thedarken.sdm/`is done, then reinstall SD Maid.
If the issue persists, check your superuser app, possibly try without root and then try again.

The log is incomplete sadly, nothing from SD Maid in it.

https://github.com/d4rken/sdmaid-public/wiki/Reporting-a-bug

If the issue remains, get another logfile from SD Maid using the instructions above (i.e. debug run log).

**Comment by elhennig** [comment]
The log is from logcat as the app fails with busybox error when trying to open the setup in order to run a debug session. A clean reinstall did not work either.
So I edited ./shared_prefs/global_preferences.xml to set general.debug.debugrun to true
I found that the cause for the error was a problem with a faulty busybox installation in the system. After fixing this, sdmaid works again.
Thanks anyways!
